scripts/check-firmware-presence-in-releases-json.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+PARENT_PATH=$( cd "$(dirname "${BASH_SOURCE[0]}")" || exit ; pwd -P )
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m' # No Color
+
+if [[ $# -ne 1 ]]
+    then
+        echo "must provide 1 argument. $# provided"
+        exit 1
+fi
+
+DEVICE=$1
+
+extract_filenames_from_json() {
+  local json_file="$1"
+
+  # Filter out 
+  #   a) 'null' from missing .url_bitcoinonly for older firmwares
+  #   b) super-old firmwares
+
+  jq -r '.[] | select(.url) | .url, .url_bitcoinonly' "$json_file" | xargs -n 1 basename | sort | uniq \
+    | grep -vFf <(cat << EOF
+null
+trezor-1.0.0.bin
+trezor-1.1.0.bin
+trezor-1.2.0.bin
+trezor-1.2.1.bin
+trezor-1.3.0.bin
+trezor-1.3.1.bin
+trezor-1.3.2.bin
+trezor-1.3.3.bin
+trezor-1.3.4.bin
+trezor-1.3.5.bin
+trezor-t1b1-1.0.0.bin
+trezor-t1b1-1.1.0.bin
+trezor-t1b1-1.2.0.bin
+trezor-t1b1-1.2.1.bin
+trezor-t1b1-1.3.0.bin
+trezor-t1b1-1.3.1.bin
+trezor-t1b1-1.3.2.bin
+trezor-t1b1-1.3.3.bin
+trezor-t1b1-1.3.4.bin
+trezor-t1b1-1.3.5.bin
```

Comment:
@prusnak We are not having those old FW binary files. Is it ok to just ignore them from this check?

We have them in `releases.json` so this check expects them in the directory, but they are not here.

Reply:
i think we should remove it from `releases.json` as they are not supported (<1.9.0)

Reply:
OK, did it here: https://github.com/trezor/data/pull/90/commits/b8000706a8c58bb2518052b3456eca5b87c2e8a4